libvast/src/format/pcap.cpp

```diff
@@ -82,7 +82,8 @@ reader::reader(const caf::settings& options, std::unique_ptr<std::istream>)
   std::string category = "vast.import.pcap";
   if (auto interface = get_if<std::string>(&options, category + ".interface"))
     interface_ = *interface;
-  input_ = get_or(options, "vast.import.read", vast::defaults::import::read);
+  input_ = std::filesystem::path{
+    get_or(options, "vast.import.read", vast::defaults::import::read)};
```

Comment:
Question for @dominiklohmann: would it be okay to include `<filesystem>` in `defaults.hpp`? Or would it blow compile time and we should do it this way?

Reply:
No for two reasons:
1. We should avoid including that file for compile-time reasons.
2. `vast.import.read` is not necessarily a path.